Sources/KlaviyoSwift/KlaviyoInternal.swift

```diff
@@ -17,16 +17,16 @@ package struct KlaviyoInternal {
     /// the apiKey (a.k.a. CompanyID) for the current SDK instance.
     /// - Parameter completion: completion hanlder that will be called when apiKey is avaialble after SDK is initilized
     package static func apiKey(completion: @escaping ((String?) -> Void)) {
-        if cancellable == nil {
-            cancellable = KlaviyoSwiftEnvironment.production.statePublisher()
-                .receive(on: DispatchQueue.main)
-                .filter { $0.initalizationState == .initialized }
-                .compactMap(\.apiKey)
-                .removeDuplicates()
-                .sink(receiveValue: {
+        cancellable = KlaviyoSwiftEnvironment.production.statePublisher()
+            .receive(on: DispatchQueue.main)
+            .filter { $0.initalizationState == .initialized }
+            .compactMap(\.apiKey)
+            .removeDuplicates()
+            .prefix(1)
```

Comment:
I think there is first too...not sure if that completes.

Reply:
![image](https://github.com/user-attachments/assets/14c161ad-61bf-473a-b094-247a0fafba32)

Reply:
Yah was just more curious what the difference was. I believe you it completes lol.

Reply:
lol I bring receipts always!